rclcpp_components/include/rclcpp_components/component_manager.hpp

```diff
@@ -131,6 +131,20 @@ class ComponentManager : public rclcpp::Node
   create_component_factory(const ComponentResource & resource);
 
 protected:
+  /// Set node options for componenet
```

Comment:
```suggestion
  /// Set node options for a component
```

Reply:
Or, if we rename the method:
```suggestion
  /// Create node options for loaded component
```

Reply:
Fixed in 5db2a22